examples/00-EDB/06_Advanced_EDB.py

```diff
@@ -132,9 +122,7 @@ def _create_ground_planes(edb, layers):
         edb.padstacks.place([xloc + "*-1", yloc + "*-1"], "GVIA", net_name="GND")
 
 ##################################################################################
-# Generate traces
-# ~~~~~~~~~~~~~~~
-# Generate and place parametric traces.
+# Draw the traces
```

Comment:
See how sometimes you have periods to conclude the sentence and sometimes you don't. We really need to be consistent.